packages/utils/src/__tests__/array.test.js

```diff
@@ -3,74 +3,115 @@
+    ['returns the minimum among the provided values (ii)', [-2, 1, 3, 10], -2],
+    ['returns `undefined` for undefined input', undefined, undefined],
+    ['returns `undefined` for null input', null, undefined],
+    ['returns `undefined` for empty array', [], undefined],
+  ];
+
+  it.each(testData)('%s', (_, input, expected) => {
+    expect(min(input)).toBe(expected);
+  });
+});
+
+describe('max', () => {
+  const testData = [
+    ['returns the maximum among the provided values (i)', [2, 3, 10], 10],
+    ['returns the maximum among the provided values (ii)', [-20, 1, 3, 10], 10],
+    ['returns `undefined` for undefined input', undefined, undefined],
+    ['returns `undefined` for null input', null, undefined],
+    ['returns `undefined` for empty array', [], undefined],
+  ];
+
+  it.each(testData)('%s', (_, input, expected) => {
+    expect(max(input)).toBe(expected);
   });
+});
 
-  describe('min', () => {
+describe('toArray', () => {
+  it('array input', () => {
+    expect(toArray(['a'])).toStrictEqual(['a']);
+    expect(toArray(['a', 'b', 'c'])).toStrictEqual(['a', 'b', 'c']);
+  });
+
+  it('non array input', () => {
+    expect(toArray('a')).toStrictEqual(['a']);
+    expect(toArray(undefined)).toStrictEqual([undefined]);
+  });
+});
+
+describe('removeAt', () => {
+  describe('throws for invalid index', () => {
     const testData = [
-      ['returns the minimum among the provided values (i)', [2, 3, 10], 2],
-      ['returns the minimum among the provided values (ii)', [-2, 1, 3, 10], -2],
-      ['returns `undefined` for undefined input', undefined, undefined],
-      ['returns `undefined` for null input', null, undefined],
-      ['returns `undefined` for empty array', [], undefined],
+      ['string', '1'],
+      ['array', [1, 2]],
+      ['negative number', -1],
+      ['+Infinity', Number.POSITIVE_INFINITY],
+      ['-Infinity', Number.NEGATIVE_INFINITY],
     ];
 
-    it.each(testData)('%s', (_, input, expected) => {
-      expect(min(input)).toBe(expected);
+    it.each(testData)('%s', (_, input) => {
+      const array = [];
+      const index = input;
+      expect(() => removeAt(array, index)).toThrow('not a positive integer');
     });
   });
 
-  describe('max', () => {
+  describe('removes the item at the specified index', () => {
     const testData = [
-      ['returns the maximum among the provided values (i)', [2, 3, 10], 10],
-      ['returns the maximum among the provided values (ii)', [-20, 1, 3, 10], 10],
-      ['returns `undefined` for undefined input', undefined, undefined],
-      ['returns `undefined` for null input', null, undefined],
-      ['returns `undefined` for empty array', [], undefined],
+      ['empty array', [[], 0], []],
+      ['non empty array, remove at start', [['a', 'b', 'c'], 0], ['b', 'c']],
+      ['non empty array, remove at middle', [['a', 'b', 'c'], 1], ['a', 'c']],
+      [
+        'non empty array, remove at middle (multilpe elements)',
```

Comment:
```suggestion
        'non empty array, remove at middle (multiple elements)',
```

Reply:
Thanks for catching this - my spell checker was disabled! 👀